### PR TITLE
test(floating-label): Add Jasmine component unit test

### PR DESCRIPTION
**Pull request by allan-chen**
Project import generated by Copybara.

### CODE REVIEW

File: packages/mdc-floating-label/test/component.test.ts [hunk 1]
```diff
@@ -22,60 +22,70 @@
  */
 
 
-import {MDCNotchedOutline} from '../../mdc-notched-outline/index';
```
Comment (abhiomkar) [inline packages/mdc-floating-label/test/component.test.ts]: it this `git copy` automatically created by Copybara? 🤔

Comment (allan-chen) [reply packages/mdc-floating-label/test/component.test.ts]: Hmm yes, this is strange

Comment (abhiomkar) [reply packages/mdc-floating-label/test/component.test.ts]: I think this PR accidentally replaced the notchedoutline test file. Can we restore it?

Comment (allan-chen) [reply packages/mdc-floating-label/test/component.test.ts]: Ah sorry, must've missed a sync - opened #5449

**Pull request merged**